core/src/main/scala/cats/Reducible.scala

```diff
@@ -226,6 +226,9 @@ abstract class NonEmptyReducible[F[_], G[_]](implicit G: Foldable[G]) extends Re
     1 + G.size(tail)
   }
 
+  override def get[A](fa: F[A])(idx: Long): Option[A] =
+    if (idx == 0L) Some(split(fa)._1) else G.get(split(fa)._2)(idx)
```

Comment:
Shouldn't this be `else G.get(split(fa)._2)(idx - 1L)`?

It's a bit funny that the only line of your PR that isn't covered by a unit test is the one with a bug in it 😜